server/api/eventType_handler.py

```diff
@@ -52,12 +52,19 @@ def create_eventType():
 
     return jsonify({"success":True,"response":"Event type succesfully created!"})
 
-@eventType_handler.route('/',methods=["GET"])
+@eventType_handler.route('/<int:id>',methods=["GET"])
 @check_token
-def get_eventType():
+def get_eventType(id):
+    user = Users.query.filter_by(id=id).first()
+    user_url = user.url
+    name = user.name
     all_eventTypes = EventTypes.query.all()
```

Comment:
I think this should be user_eventType = user.eventType instead of all the eventTypes in db

Reply:
Oh, I was planning to change that part but forgot, I'll be able to change it later tonight